server/src/main/java/com/lunchtime/models/MenuItemDish.java

```diff
@@ -47,11 +40,11 @@
     public MenuItemDish() {
```

Comment:
Could you place methods after fields?
https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents

Reply:
Fixed